UniConnectApp/app/src/test/java/com/example/uniconnect/UniConnectUnitTest.kt

```diff
@@ -29,16 +27,16 @@ import java.util.concurrent.TimeUnit
 class UniConnectUnitTest {
 
     //lateinit var mvm : MainViewModel
-
+//
     //@MockK
     //lateinit var mockUniService : UniversityService
 
     @get:Rule
     var rule: TestRule = InstantTaskExecutorRule()
     lateinit var universityService: IUniversityService
     var allUniversities : List<University>? = ArrayList<University>()
-
-    private val mainThreadSurrogate = newSingleThreadContext("UI thread")
+    val thread = "UI thread"
+    private val mainThreadSurrogate = newSingleThreadContext(thread)
```

Comment:
This doesn't appear to add any value... over creating a new line of code... unless the variable thread is used elsewhere.